src/beacon-api/app/api/v1/collocation.py

```
@@ -130,6 +134,151 @@ async def get_inlab_collocation(
         raise HTTPException(status_code=500, detail="Internal Server Error")
 
 
+# ── Inlab Batch CRUD ───────────────────────────────────────────────
+
+@router.post("/inlab/batch", response_model=InlabBatchDetailResponse)
+async def create_inlab_batch(
+    payload: InlabBatchCreate,
+    authorization: str = Header(...),
+    db: Session = Depends(get_db),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)